packages/docusaurus-plugin-content-blog/src/blogFrontMatter.ts

```diff
@@ -39,7 +39,6 @@ const BlogPostFrontMatterAuthorSchema = Joi.object({
   .rename('image_url', 'imageURL', {alias: true});
 
 export type BlogPostFrontMatter = {
-  /* eslint-disable camelcase */
```

Comment:
ESLint doesn't seem to report `camelcase` for types anymore. I tried with https://typescript-eslint.io/rules/naming-convention to keep the same behavior but we have much more places where we don't conform to any rule